Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV.Tests/ExcelConvertTests.cs

```diff
@@ -13,93 +13,141 @@ public class ExcelConvertTests
     private readonly Input _input = new();
     private readonly Options _options = new();
 
-    // Cat image in example files is from Pixbay.com. It is licenced in CC0 Public Domain (Free for commercial use, No attribution required).
-    // It is uploaded by Ben_Kerckx https://pixabay.com/en/cat-animal-pet-cats-close-up-300572/
+    // Cat image in example files is from Pixbay.com. It is licenced in CC0 Public Domain (Free for commercial use, No attribution required). It is uploaded by Ben_Kerckx https://pixabay.com/en/cat-animal-pet-cats-close-up-300572/
```

Comment:
Lets not have such long lines. Ideally code and comments should fit into range of 120-160 chars. This line is too long.